core/js_errors.rs

```diff
@@ -9,28 +9,16 @@
 //   console.log(err.stack);
 // It would require calling into Rust from Error.prototype.prepareStackTrace.
 
-use crate::any_error::ErrBox;
-use serde_json;
-use serde_json::value::Value;
+use crate::ErrBox;
+use rusty_v8 as v8;
+use std::convert::TryFrom;
+use std::convert::TryInto;
 use std::error::Error;
 use std::fmt;
-use std::str;
 
 #[derive(Debug, PartialEq, Clone)]
-pub struct StackFrame {
-  pub line: i64,   // zero indexed
-  pub column: i64, // zero indexed
-  pub script_name: String,
-  pub function_name: String,
-  pub is_eval: bool,
-  pub is_constructor: bool,
-  pub is_wasm: bool,
-}
-
-#[derive(Debug, PartialEq, Clone)]
-pub struct V8Exception {
+pub struct JSError {
```

Comment:
We often have confusion with all the different types of errors - it would be helpful if you added a triple slash comment explaining exactly where this error comes into play. 

Maybe something like "JSError represents exceptions coming from V8, with stack frames and line numbers. This structure is modified by source maps, and ultimately used to display colorful stack traces."